### ISSUE TITLE
Making GPU Utilization and Memory line graphs

### ISSUE THREAD

**Issue by jakirkham** [rapidsai/jupyterlab-nvdashboard#26]
Currently GPU Utilization and Memory plots are bar plots. It would nice to turn these into line graphs. This would let us track usage at present and moments prior.

**Comment by jacobtomlinson** [comment]
These are available in the "GPU Resources" panel. Does that meet your needs or are you after something else?

![image](https://user-images.githubusercontent.com/1610850/66553808-bb58b580-eb43-11e9-9081-1eeb7038053d.png)

**Comment by jakirkham** [comment]
Right. I'm referring to the other two dashboards with the same name.

**Comment by jacobtomlinson** [comment]
Yeah sure. I guess my point is both formats are currently available. However the bars are only available individually and the lines are only available in a larger dashboard.

I find bars easier to glance at but lines better for analysing what is happening. Therefore I would like both to be available. 

We could make individual bars available too. Or even add an option to switch between them somehow within one dashboard?

**Comment by rjzamora** [comment]
I have been meaning to add line-plot versions of utilization and memory (especially utilization).  My only hesitation was to avoid adding too many individual dashboards.  I do like the idea of allowing the user to toggle between different representations within the same dashboard
